plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/project/ManageProjectsWizardPage.java

```diff
@@ -0,0 +1,291 @@
+				}
+				final List<IProject> oldProjects = pageModel.getProjects();
+				final List<IProject> newProjects = new ArrayList<IProject>(oldProjects);
+				newProjects.remove(project);
+				DeleteResourceJob job = new DeleteResourceJob(project) {
+
+					@Override
+					protected IStatus doRun(IProgressMonitor monitor) {
+						IStatus status = super.doRun(monitor);
+						if(status.isOK()) {
+							ConnectionsRegistrySingleton.getInstance().fireConnectionChanged(
+									pageModel.getConnection(), 
+									ConnectionProperties.PROPERTY_PROJECTS, 
+									oldProjects, 
+									newProjects);
+						}
+						return status;
+					}
+					
+				};
+				try {
+					org.jboss.tools.common.ui.WizardUtils.runInWizard(
+							job, job.getDelegatingProgressMonitor(), getContainer(), dbc);
+				} catch (InvocationTargetException ex) {
+					OpenShiftUIActivator.getDefault().getLogger().logError(NLS.bind("Could not delete OpenShift project {0}", project.getName()), ex);
+				} catch (InterruptedException ex) {
+					OpenShiftUIActivator.getDefault().getLogger().logError(NLS.bind("Could not delete OpenShift project {0}", project.getName()), ex);
+				}
+			}
+		};
+	}
+
+	private SelectionListener onRefresh(final DataBindingContext dbc) {
+		return new SelectionAdapter() {
+
+			@Override
+			public void widgetSelected(SelectionEvent e) {
+				refreshModel(dbc);
+			}
+		};
+	}
+
+	@Override
+	protected void setupWizardPageSupport(DataBindingContext dbc) {
+		ParametrizableWizardPageSupport.create(IStatus.ERROR | IStatus.CANCEL, this, dbc);
+	}
+
+	protected TableViewer createTable(Composite tableContainer) {
+		Table table =
+				new Table(tableContainer, SWT.BORDER | SWT.FULL_SELECTION | SWT.V_SCROLL | SWT.H_SCROLL);
+		table.setLinesVisible(true);
+		table.setHeaderVisible(true);
+		ICellToolTipProvider<IProject> cellToolTipProvider = new ICellToolTipProvider<IProject>() {
+
+			@Override
+			public String getToolTipText(IProject object) {
+				return object.getDescription();
+			}
+
+			@Override
+			public int getToolTipDisplayDelayTime(IProject object) {
+				return 0;
+			}
+		};
+		
+		this.viewer = new TableViewerBuilder(table, tableContainer)
+				.contentProvider(new ArrayContentProvider())
+				.column(new IColumnLabelProvider<IProject>() {
+
+					@Override
+					public String getValue(IProject project) {
+						return project.getName();
+					}
+				})
+				.cellToolTipProvider(cellToolTipProvider)
+				.name("Name").align(SWT.LEFT).weight(1).minWidth(75).buildColumn()
+				.column(new IColumnLabelProvider<IProject>() {
+
+					@Override
+					public String getValue(IProject project) {
+						return project.getDisplayName();
+					}
+				})
+				.cellToolTipProvider(cellToolTipProvider)
+				.name("Display Name").align(SWT.LEFT).weight(2).minWidth(100).buildColumn()
+				.buildViewer();
+
+		return viewer;
+	}
+
+	private void refreshModel(final DataBindingContext dbc) {
+		try {
+			org.jboss.tools.common.ui.WizardUtils.runInWizard(
```

Comment:
looks like the full-package name isnt required here any more since there's no WizardUtils in openshift any more?